models/monitored_operator.go

```diff
@@ -48,6 +48,9 @@ type MonitoredOperator struct {
 
 	// Positive number represents a timeout in seconds for the operator to be available.
 	TimeoutSeconds int64 `json:"timeout_seconds,omitempty"`
+
+	// Version of the operator.
+	Version string `json:"version,omitempty"`
```

Comment:
How would OCS operator version be determined?

Reply:
The code is here: https://github.com/openshift/assisted-installer/pull/284/files#diff-1da579fda3bc689940661d48dd744a46f23824270427207cfae4f8c7448eddd2R739

Equivalent of:
```bash
oc get csv csvname -n csvnamespace  -o jsonpath='{.spec.version}'
```

Reply:
It would automatically update the operator version from csv, right?

Reply:
Correct.

Reply:
thanks